Flowpack/media-ui
# FEATURE: asset collection editing

TODO:
- [x] support editing tags belonging to the collection
- [x] give some header to the inspector so it would be visible which kind of entity we are currently editing
- [x] refactor inspector components


**What I did**

Support editing asset collection title in the inspector.

**How to verify it**

Try to edit a title of some asset collection. Don't forget to check optimistic updates.
